### PR TITLE
Deployment Guide and Trey Research updates to align documentation to Unified Deployment UI

### PR DESCRIPTION
**Pull request by juazasan**
Updates included:
- Deployment Guide for hub & spoke and single platform subscription scenario.
- TR Readme updated to reference both the Unified UI and the Deployment Guide.

### CODE REVIEW

**Review by rjfmachado — APPROVED** [review]

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    
+    In this tutorial, you will not enable this feature. Please note you can [enable DNS Proxy feature](https://docs.microsoft.com/en-us/azure/firewall/dns-settings) at any moment.
+  
+    Set **Enable Azure Firewall as a DNS proxy** to **No**.
+
+    ![networkTab-fwDNSProxy](./media/clip_image036b-7-singlesubscription.png)
+
+  - **Select Azure Firewall tier**: choose the right SKU based on your requirements. See [Azure Firewall features](https://docs.microsoft.com/en-us/azure/firewall/features) and [Azure Firewall Premium features](https://docs.microsoft.com/en-us/azure/firewall/premium-features) for further details.
+
+    In this tutorial, you will deploy an Azure Firewall Standard. Please note you can [upgrade to Azure Firewall Premium](https://docs.microsoft.com/en-us/azure/firewall/premium-migrate) later on if needed provided you can afford some downtime.
```
Comment (victorar) [inline docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:187]: The only challenge I see with this recommendation, is that if customers deploy AZFW Std in prod, and later they realize they need the Premium features (like IDPS), then they will experience an outage in their environment when moving from Std to Premium.

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+  
+    Set **Select Azure Firewall tier** to **Standard**:
```
Comment (victorar) [inline docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:189]: Same as previous comment.

**Review by victorar — APPROVED** [review]
LGTM! I left a couple of comments for consideration.

**Review by kiwibayer — COMMENTED** [review]
@juazasan , interested in your opinion. 
I would change the layout slightly to show that these are explicit steps that should be done one after one by adding numbers before the Headers ... 
e.g. 1. Pre-Requisites 
2. Launch the Deployment Experience 
3. Deployment location 
...

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md (hunk 1 above)
Comment (juazasan) [reply docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:187]: The wording I have was supposed to already warn the customer about potential downtime when doing the upgrade: "later on if needed **provided you can afford some downtime**."

Will try to re-write to make it clearer.

File: docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md (hunk 2 above)
Comment (juazasan) [reply docs/wiki/Deploying-Enterprise-Scale-BasicSetup.md:189]: The wording I have was supposed to already warn the customer about potential downtime when doing the upgrade: "later on if needed provided you can afford some downtime."

Will try to re-write to make it clearer.

**Review by kiwibayer — APPROVED** [review]

**Pull request merged**